azure-maven-plugin-lib/pom.xml

```diff
@@ -142,6 +142,22 @@
             <groupId>org.dom4j</groupId>
             <artifactId>dom4j</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Why we need to declare dependency for `asm` here?

Reply:
updated.